recipes/default.rb

```diff
@@ -1,5 +1,6 @@
 # Recipes controlled by attributes
 include_recipe 'bubble::packages' if node['bubble']['packages']
+group node['bubble']['group_name']
```

Comment:
@borisroman This group is used in several recipes as owner, but nowhere the group is created. Except (default) when the users recipe is included. The example databag "happens" to have the same groupname filled in with the user, hence created.

When running this on a "vanilla" centos box, and not using the users recipe, it will fail.